extensions/sql-migration/src/api/sqlUtils.ts

```diff
@@ -127,6 +127,32 @@ export interface LoginTableInfo {
 	status: string;
 }
 
+export async function getSourceConnectionProfile(): Promise<azdata.connection.ConnectionProfile> {
+	return await azdata.connection.getCurrentConnection();
+}
+
+export async function getSourceConnectionId(): Promise<string> {
+	return (await getSourceConnectionProfile()).connectionId;
+}
+
+export async function getSourceConnectionServerInfo(): Promise<azdata.ServerInfo> {
+	return await azdata.connection.getServerInfo((await getSourceConnectionProfile()).connectionId);
```

Comment:
can these methos be updated to replace (await getSourceConnectionProfile()).connectionId with  getSourceConnectionId etc

Reply:
thanks, good catch